src/main/java/WaterPokemon.java

```diff
@@ -0,0 +1,50 @@
+public class WaterPokemon extends Pokemon{
+    private String propel;
+    private int attackDistance;
+
+//    constructor
+    public WaterPokemon(String name, int level,String propel, int attackDistance) {
+        super(name, level);
+        super.setPokemonType("Water");
+        this.propel=propel;
+        this.attackDistance=attackDistance;
+    }
+
+//    methode 1
+    public void usePropel(){
+        System.out.println(super.getName()+" uses its "+propel+" to move around in the water.");
+    }
+
+//    methode 2
+    public void attack(){
```

Comment:
Dit klinkt niet als een methode die specifiek voor water pokemons geldt, maar iets wat voor alle pokemons geldt. Deze methode past daarom beter in de super klasse.